### PR TITLE
"Fixed" Platoon integration

### PR DESCRIPTION
**Pull request by mgermain**
It was already working with the last version but I removed some unnecessary code for operations that are now done seamlessly directly in Platoon.

### CODE REVIEW

**Comment by nouiz** [comment]
blocks have its own dispatcher of process. Should we remove it? Can you compare if the one in platoon cover all what the one here do?

File: blocks_extras/extensions/synchronization.py [hunk 1]
```diff
@@ -137,15 +137,6 @@ def seed(self):
     def init_shared_params(self, parameters):
         super(SynchronizeWorker, self).init_shared_params(
             parameters, self.sync_rule)
-        if self.is_main_worker:
```
Comment (rizar) [inline blocks_extras/extensions/synchronization.py]: We should go through all the documentation and make sure that it does not mention this initialization trick

Comment (rizar) [inline blocks_extras/extensions/synchronization.py]: E.g., line 105 seems to be obsolete with these changes.

**Comment by rizar** [comment]
@nouiz , your comment leaves me puzzled. All we have in Blocks is a script to run the controller, and it is assumed that the workers are launched manually.

**Comment by nouiz** [comment]
I taught that script was also starting the worker. If you think it is still
needed, we can keep it, but if we don't need it, I think we should clean
this up now in this PR if possible.

**Comment by mgermain** [comment]
No, they do need the controller.

**Comment by rizar** [comment]
Should I merge this, or do you think that something else should be done here?

**Comment by mgermain** [comment]
Looks good to me.

On Fri, Feb 19, 2016 at 10:34 AM, Dzmitry Bahdanau <notifications@github.com

**Pull request merged**